src/08-accessing.js

```diff
@@ -103,7 +115,10 @@ function getFirstPhoneNumber(person) {}
  *  getLastPhoneNumber(person);
  *  //> 7185558611
  */
-function getLastPhoneNumber(person) {}
+function getLastPhoneNumber(person) {
+  let x = person.numbers.shift(-1)
```

Comment:
Take another look at how `.shift()` works under the hood.  What do we call it with?